### PR TITLE
Fix styles

### PR DESCRIPTION
**Pull request by btzr-io**
- PR https://github.com/buttercomponents/butter-component-builder/pull/3
- Remove `postcss-loader`

### CODE REVIEW

**Comment by btzr-io** [comment]
this will fix other react errors / warnings from `butter-component-settings` : 
https://github.com/buttercomponents/butter-component-settings/pull/16

File: postcss.config.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module.exports = {
+}
```
Comment (xaiki) [inline postcss.config.js]: this should go

Comment (btzr-io) [reply postcss.config.js]: ok

**Comment by btzr-io** [comment]
that should fix the current issue with `butter-desktop/xaiki/react`: https://github.com/buttercomponents/butter-component-builder/pull/5/commits/9dae7551075d3a9beba8ebb74f763071430b5a33

**Comment by xaiki** [comment]
changing the indent makes it very hard to review on github =/

**Comment by btzr-io** [comment]
yeah sorry about that 😛 ^^
(anyway it was just last commit and only changed `externals`)

**Pull request merged**